front/src/modules/ui/utilities/pointer-event/hooks/useListenClickOutside.ts

```diff
@@ -85,12 +85,13 @@ export const useListenClickOutsideByClassName = ({
   callback: () => void;
 }) => {
   useEffect(() => {
-    const handleClickOutside = (event: MouseEvent) => {
+    const handleClickOutside = (event: MouseEvent | TouchEvent) => {
```

Comment:
Since mobile version handles touchevents, we might need to do more work here...?

Reply:
Yes see touchend like the function above